src/transformers/models/cvt/configuration_cvt.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+# Copyright 2022 AnugunjNaman and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Cvt model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+CVT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "msft/cvt": "https://huggingface.co/msft/cvt/resolve/main/config.json",
```

Comment:
```suggestion
    "microsoft/cvt": "https://huggingface.co/microsoft/cvt/resolve/main/config.json",
```
All checkpoints should be placed under the "microsoft" organization (I can help with that).